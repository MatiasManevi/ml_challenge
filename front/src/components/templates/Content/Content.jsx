import React from 'react';
import Container from '@material-ui/core/Container';
import Grid from '@material-ui/core/Grid';

import './Content.scss';

const Content = ({ children }) => {
	return (
		<div className="content">
			<Container>
				<Grid container justify="center">
					<Grid item xs={10}>
						{children}
					</Grid>
				</Grid>
			</Container>
		</div>
	);
};

export default Content;
