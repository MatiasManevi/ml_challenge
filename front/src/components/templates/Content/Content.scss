@import 'assets/scss/variables';

.content {
	height: 100%;
	padding-bottom: $space-42;
}
