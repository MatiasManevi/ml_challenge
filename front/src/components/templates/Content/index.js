export { default } from 'components/templates/Content/Content';
