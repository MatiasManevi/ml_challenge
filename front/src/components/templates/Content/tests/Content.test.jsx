import React from 'react';
import { shallow } from 'enzyme';

import Content from '../Content';

describe('Content', () => {
	it('should match snapshot', () => {
		const children = 'Some text';
		const wrapper = shallow(<Content>{children}</Content>);
		expect(wrapper).toMatchSnapshot();
	});
});
