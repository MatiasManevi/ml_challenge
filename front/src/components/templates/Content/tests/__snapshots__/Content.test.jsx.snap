// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should match snapshot 1`] = `
<div
  className="content"
>
  <WithStyles(ForwardRef(Container))>
    <WithStyles(ForwardRef(Grid))
      container={true}
      justify="center"
    >
      <WithStyles(ForwardRef(Grid))
        item={true}
        xs={10}
      >
        Some text
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Container))>
</div>
`;
